client/my-sites/email/email-management/gsuite-user-item/style.scss

```diff
@@ -10,6 +10,7 @@
 	.gsuite-user-item__email {
 		float: left;
 		line-height: 1.7;
+		font-weight: 600;
 	}
```

Comment:
I know it's not part of coding conventions, but I find rules to be easier to scan when they are kept ordered alphabetically.